File: Andy/src/level1/javascript/StringPYCount.js
//프로그래머스 Level 1, 문자열 내 p와 y의 개수
function solution(s){
    var p = s.match(/P/ig);   
    var y = s.match(/Y/ig);
    return ((p?p.length:0) == (y?y.length:0));
}

File: Andy/src/level1/java/StringPYCount.java
package level1.java;

//프로그래머스 Level 1, 문자열 내 p와 y의 개수
public class StringPYCount {
    boolean solution(String s) {
        if( s.replaceAll("[^pP]","").length() == s.replaceAll("[^yY]","").length() ) return true;
        return false;
    }
}
